main/lsp/requests/definition.cc

```diff
@@ -132,6 +132,23 @@ unique_ptr<ResponseMessage> DefinitionTask::runRequest(LSPTypecheckerDelegate &t
                     start = start->secondary.get();
                 }
             }
+        } else if (fileIsTyped && resp->isKeywordArg()) {
+            auto kw = resp->isKeywordArg();
```

Comment:
I think we have fancy C++ syntax for this, something like:

```suggestion
        } else if (auto kw = resp->isKeywordArg(); fileIsTyped && kw) {
```

Reply:
went with a slightly different approach, [b91f396](https://github.com/sorbet/sorbet/pull/9428/commits/b91f3969baf9cea33a40eb07ce15eb80e8e92eec)